nautobot_circuit_maintenance/handle_notifications/handler.py

```diff
@@ -71,13 +71,12 @@ def create_circuit_maintenance(
                 )
         else:
             note_entry, created = Note.objects.get_or_create(
-                maintenance=circuit_maintenance_entry,
-                title=f"Nonexistent circuit ID {circuit.circuit_id}"[:MAX_NOTE_TITLE_LENGTH],
-                comment=(
+                assigned_object_id=circuit_maintenance_entry.id,
+                assigned_object_type=ContentType.objects.get_for_model(CircuitMaintenance),
```

Comment:
I forget, can you assign directly to the GFK instead (`assigned_object=circuit_maintenance_entry`) or is that a no-go here?

Reply:
I think I tried, and I got some issue